crates/tauri/src/menu/menu.rs

```diff
@@ -177,10 +177,11 @@ impl<R: Runtime> Menu<R> {
       HELP_SUBMENU_ID,
       "Help",
       true,
-      &[
-        #[cfg(not(target_os = "macos"))]
-        &PredefinedMenuItem::about(app_handle, None, Some(about_metadata))?,
-      ],
+      &[&PredefinedMenuItem::about(
```

Comment:
On macOS the About stuff is already in the first menu item (the one with the appname as label)

Reply:
Electron's default menubar has Electron's own links in there, that's an option

<img width="357" height="153" alt="image" src="https://github.com/user-attachments/assets/8522119f-6afc-4f8c-9df0-aeb17e99dcbe" />

Reply:
i agree that it looks weird without anything (at least as far as i can remember, didn't see check for ages)

Reply:
It actually doesn't look _that_ weird on macos 26 anymore with all that added padding. Can we perhaps still revert this part? I'm completely on board with the hotfix of the actual issue so i'd like to get it in asap but i think what we put into the help menu needs a bit more thinking since right now it's only added for visuals.

> Electron's default menubar has Electron's own links in there, that's an option

Actually not a fan of that at all since it basically forces everyone to create their own menu even if they want just the default (cause they actually don't want a menu at all in the context of windows/linux)